pkg/validate/validate.go

```diff
@@ -77,9 +77,7 @@ func validateUpdateContainerService(cs, oldCs *api.OpenShiftManagedCluster, exte
 func validateProperties(p *api.Properties, externalOnly bool) (errs []error) {
 	errs = append(errs, validateProvisioningState(p.ProvisioningState)...)
 	errs = append(errs, validateOrchestratorProfile(p.OrchestratorProfile, externalOnly)...)
-	if !externalOnly {
-		errs = append(errs, validateFQDN(p)...)
-	}
+	errs = append(errs, validateFQDN(p)...)
```

Comment:
you won't need externalOnly as an argument to validateProperties any more - can you remove it from the functions where it's not needed?

Reply:
`errs = append(errs, validateOrchestratorProfile(p.OrchestratorProfile, externalOnly)...)` for router profile?

Reply:
Or we dont want to validate this via `externalOnly` too? Missing logic in my head for this

Reply:
bah, ignore me, sorry